### DISCUSSION TITLE
Cannot submit a rating of 0 - Rating field

### DISCUSSION THREAD

**Discussion by billymedia** [solspace/craft-freeform#532] [Feature Requests]
When i add a rating field to a form (ajax enabled), have it set to not be required, and submit the form in the front end with no star selected, i can see from the dev console network request that it sees an empty rating value as an error. This makes it impossible to submit a value of zero. As well as this, it doesnt add an error css class.

**Steps to reproduce**

1. Create a form with a non mandatory rating field
2. Submit the form with no star selected

**Expected behavior**
Save the form submission with a star rating of zero

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.7.55.2
- Freeform Version: 3.13.19
- Freeform Edition: Pro

**Comment by kjmartens** [comment]
Unfortunately, Rating fields are required to have a value of at least `1`. I will note this for the feature requests lists to consider supporting `0` in the future. 🙂
